src/TableTrait.php

```
@@ -75,6 +84,149 @@ public function tableAssertColumns(string $selector, TableNode $table): void {
+        }
+        if ($match) {
+          $found = TRUE;
+          break;
+        }
+      }
+
+      if (!$found) {
+        throw new ExpectationException(sprintf('Row %d with values [%s] not found in table "%s".', $row_index + 1, implode(', ', array_values($expected_row)), $selector), $this->getSession()->getDriver());
+      }
+    }
+  }
```

PHPMD (2.15.0)

[warning] The method tableAssertRows() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)